src/utils/validators.js

```diff
@@ -26,6 +26,13 @@ export const gatewayHostValidator = (value) => {
         .test(value);
 };
 
+export const sipAccountValidator = (value) => {
+  if (typeof value === 'undefined' || value === null || value === '') {
+    return true;
+  }
+  return /^(sips?:)?([\w\-\.!~\*\(\)&=+\$,;\?\/]+)(?:@([\w\-\.]+)(?::(\d{3,5}))?)?$/i.test(value);
```

Comment:
I guess, we need also to escape PLUS '+' character in the second group for `SIP-userinfo` validation to avoid regex's '+' control character misunderstanding